reana_db/alembic.ini

```diff
@@ -0,0 +1,56 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = %(here)s/alembic
+
+# version location specification; this defaults
+# to reana_db/alembic/versions.  When using multiple version
+# directories, initial revisions must be specified with --version-path
+version_locations = %(here)s/alembic/versions
+
+[post_write_hooks]
+# post_write_hooks defines scripts or Python functions that are run
+# on newly generated revision scripts.  See the documentation for further
+# detail and examples
+
+# format using "black" - use the console_scripts runner, against the "black" entrypoint
```

Comment:
When I create a revision I get a `FAILED`, not sure if it comes from black, have you seen it?

```console
root@reana-server-66d5649989-mqcbw:/code# reana-db alembic revision -m 'add lots of things'
  Generating /code/modules/reana-db/reana_db/alembic/versions/270a3a1b9e85_add_lots_of_things.py ...  done
  Running post write hook "black" ...
  FAILED
```

Reply:
No, I didn't see this error and I tested thoroughly the revision creation. The config is the default one, so if you have format on-save I don't see why this would be happening, have you modified the revision template?

Reply:
It was failing because I didn't have Black installed in the server pod. This dependency comes from `pytest-reana` which is [`tests_require`](https://setuptools.readthedocs.io/en/latest/setuptools.html) so I guess it's not installed on deploy